json-rpc/tests/integration_test.rs

```diff
@@ -205,7 +205,18 @@ fn create_test_cases() -> Vec<Test> {
                             "compliance_key": "",
                             "expiration_time": 18446744073709551615_u64,
                             "human_name": "moneybags",
-                            "preburn_balances": [],
+                            "preburn_balances": [
+                                {
+                                    "amount": 0_u64,
+                                    "currency": "XUS"
+                                }
+                            ],
+                            "preburn_queues": [
+                                {
+                                    "preburns": [],
+                                    "currency": "XUS",
+                                }
+                            ],
```

Comment:
Could we add a test that produces non-zero preburn_balances info? it's fine if we could not produce preburn_queues yet, but from my understanding, we at least should get same result in preburn_queues with preburn_balances, before we enable the feature.
Not sure whether we can have test with feature enabled, it would be great to have it too.

The diff looks solid, but I trust more on test result than my eyes.

Reply:
Sounds good. I'll work on adding that test right now. We should indeed get the same result in both of those fields. 

Currently we can only create preburn queues (so with the feature enabled) since it is running on the head of the framework.

Reply:
Just added the test. Added it as another integration test, and made sure that the `preburns_balances` value is equal to the sum of the preburn requests in the preburn queue.